pages/common/xml-edit.md

```diff
@@ -0,0 +1,28 @@
+# xml edit
+
+> Edit an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Delete elements matching an XPATH from an XML document:
+
+`xml edit --delete {{"XPATH1"}} {{path/to/input.xml|URI}}`
+
+- Move an element node of an XML document from XPATH1 to XPATH2:
+
+`xml edit --move {{"XPATH1"}} {{"XPATH2"}} {{path/to/input.xml|URI}}`
+
+- Rename all attributes named "id" to "ID":
+
+`xml edit --rename {{"//*/@id"}} -v {{"ID"}} {{path/to/input.xml|URI}}`
+
+- Rename XML elements of "table" named "rec" to "record":
+
+`xml edit --rename {{"/xml/table/rec"}} -v {{"record"}} {{path/to/input.xml|URI}}`
+
+- Update the value of XML table record ID=3 to 5:
```

Comment:
So, every `ID` which is 3 will be 5, right? I think that's a bit clearer:
```suggestion
- Update the value of the XML table record `ID=3` to `ID=5`:
```

Reply:
sure

Reply:
- Update the XML table record with "id=3" to the value "id=5":